tests/radshockMG.in

```diff
@@ -0,0 +1,21 @@
+# *****************************************************************
+# Problem size and geometry
+# *****************************************************************
+geometry.prob_lo     =  0.0  0.0  0.0 
+geometry.prob_hi     =  0.01575  0.001575  1.0
+geometry.is_periodic =  0    1    1
+
+# *****************************************************************
+# VERBOSITY
+# *****************************************************************
+amr.v              = 0       # verbosity in Amr
+
+# *****************************************************************
+# Resolution and refinement
+# *****************************************************************
+amr.n_cell          = 128 4 4
+amr.max_level       = 0     # number of levels = max_level + 1
+amr.blocking_factor = 4     # grid size must be divisible by this
```

Comment:
GPU performance might be improved if you set `amr.max_grid_size_x = 128`, `amr.max_grid_size_y = 8` and `amr.max_grid_size_z = 8`, and `amr.blocking_factor = 8`.